src/gui/mzroll/mainwindow.cpp

```diff
@@ -2458,6 +2465,29 @@ int MainWindow::loadMetaCsvFile(string filename){
 
         }
 
+        qreal *redF, *greenF, *blueF, *alpha;
+        if (header.count("color") && header["color"] < N) {
+
+            string colorName = fields[header["color"]];
+            redF = new qreal(-1);
+            greenF = new qreal(-1);
+            blueF = new qreal(-1);
+            alpha = new qreal(-1);
+            QColor color;
+            
+            if(!colorName.empty()) {
```

Comment:
Minor suggestion: instead of checking the truth condition and then proceeding with (more than 2 or 3) lines of code, I would just `return` or `continue` if the condition is opposite of what I want it to be. You will save on one level of indentation:
```c++
if (colorName.empty())
    continue;

// followed by code that should execute if there was a non-empty color name
.
.
.
```

Does that make sense? You need not change it now, but just something to keep in mind for future.